Composer/packages/extensions/visual-designer/src/components/nodes/templates/FormCard.tsx

```diff
@@ -95,47 +96,35 @@ export const FormCard: FunctionComponent<NodeProps> = ({
           height: contentHeight,
         }}
       >
-        <div
-          css={{
-            fontWeight: 400,
-            paddingLeft: '5px',
-            margin: '3px 5px',
-            fontSize: '14px',
-            lineHeight: '19px',
-            display: 'flex',
-            alignItems: 'center',
-          }}
-        >
-          {icon && icon !== ElementIcon.None && (
-            <div
-              css={{
-                width: 16,
-                height: 16,
-                display: 'flex',
-                alignItems: 'center',
-                justifyContent: 'center',
-                marginRight: '5px',
-              }}
-            >
-              <Icon icon={icon} color={iconColor} size={iconSize || 16} />
-            </div>
-          )}
+        {label && (
```

Comment:
Is it necessary to change this file? There will be conflicts with another PR potentially.

Reply:
Fixed conflict. This is beacause there is no lable in `setProperties` node. there is a empty div to take up space if without the condition.